app/models/team.rb

```diff
@@ -3,4 +3,12 @@ class Team < ActiveRecord::Base
   has_many :income
   has_many :bonus_credit
   has_many :public_relation
+
+  def self.countries
+    self.pluck(:team_name).delete_if {|x| x == "Aliens"}
+  end
+
+  def self.all_minus_aliens
+    self.all.where.not(team_name: "Aliens")
```

Comment:
This will work for now, but I think we should probably have something else in place for the future where we a) might use a different name or b) might have multiple alien factions. 

That's probably an over-optimisation on my part, though.